.github/actions/build-validate-artifact/action.yml

```diff
@@ -1,12 +1,18 @@
 name: Build and Validate Artifact
 description: Reusable action to build and validate the component artifact
+inputs:
+  output-artifact:
+    description: 'If provided, the action will output the artifact to the build directory'
+    required: false
 runs:
   using: composite
   steps:
-    - name: Setup workspace
-      uses: "./.github/actions/bootstrap"
     - name: Build & Validate Artifact
       shell: bash
       id: build-validate-artifact
       run: |
-        cd utils && yarn build-validate-quickstart-artifact
+        cd utils && yarn build-validate-quickstart-artifact ${{ inputs.output-artifact && '--output-artifact' || '' }}
```

Comment:
silly question, does this actually supply `--output-artifact`?  if `inputs.output-artifact` is truthy?

in other words, would want a ternary here?

Reply:
Fun fact! You can't ternary in gh actions syntax :)